### DISCUSSION TITLE
你好，能不能增加一个网盘呀

### DISCUSSION THREAD

**Discussion by kpzy** [AlistGo/alist#4463] [General]
我是程序小白，不知道怎么添加，不知道有没有大佬可以加上
http://www.ys168.com/
就是这个永硕网盘
很古老，但是很稳定，应该是国内存活时间最长的网盘了
希望可以加上

**Comment by orangejx** [comment]
可能性比较低, 而且是没在永硕官网找到相关的开发者文档, 永硕公布开发文档的可能性几乎为零, 要是公布了 API 接口, 就相当于免费版本的广告显示为摆设了.

此外, 永硕免费版的计划不论是空间容量, 下载量, 单文件大小限制, 以及目录数量都不是特别出色. 
[http://www.ys168.com/pricing/](http://www.ys168.com/pricing/)
